po/ja.po

```diff
@@ -1528,30 +1528,39 @@ msgid ""
 "give a brief overview of what Cargo is and how it fits into the wider ecosystem\n"
 "and how it fits into this training."
 msgstr ""
+"Rustを学び始めると、まもなくRustエコシステムで広く使われているビルドシステム兼パッケージマネージャ"
+"である[Cargo](https://doc.rust-lang.org/cargo/)という標準ツールに出会います。ここでは、Cargoの概要"
+"や使用方法、そして本講座における重要性について簡単に説明します。"
 
 #: src/cargo.md:8
 msgid "## Installation"
-msgstr ""
+msgstr "## インストール"
 
 #: src/cargo.md:10
 msgid "### Rustup (Recommended)"
-msgstr ""
+msgstr "### Rustup (推奨)"
 
 #: src/cargo.md:12
 msgid ""
 "You can follow the instructions to install cargo and rust compiler, among other standard ecosystem "
 "tools with the [rustup][3] tool, which is maintained by the Rust Foundation."
 msgstr ""
+"[rustup][3]ツールを使用して、cargoやrustコンパイラなどの標準エコシステムツールをインストールしま"
+"す。RustupはRust Foundationによってメンテナンスされています。[rustup][3]ツールを使用して、cargoや"
+"rustコンパイラなどの標準エコシステムツールをインストールします。RustupはRust Foundationによってメン"
```

Comment:
It seems that the same sentences are repeated twice

Reply:
darn, ty